src/redux/books/books.js

```diff
@@ -1,40 +1,56 @@
 /* eslint-disable */
+import { createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
- [ ] I can see that you imported  `createAsyncThunk` but you did not use it . Kindly note that the project requires that `createAsyncThunk`  is used in the project for creating thunk action creators:

e.g. 
To get books :
```
export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```

KIndly do this for other action creators in your app 👍🏽

Reply:
Thank! Changes applied